dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetShippingMethodsOnCartTest.php

```diff
@@ -161,6 +393,10 @@ private function prepareMutationQuery(
 QUERY;
     }
 
+    private function addShippingMethodToQuote(Quote $quote)
+    {
```

Comment:
This is unused private method.

Reply:
Yes, I will work on refactoring and use this method.